src/server/schemas/sharedApiSchemas.ts

```diff
@@ -332,3 +332,10 @@ export const walletDetailsSchema = Type.Object({
     Type.Null(),
   ]),
 });
+
+export const contractSubscriptionResponseSchema = Type.Object({
```

Comment:
Can you name this `contractSubscriptionConfigurationSchema` to more clearly differentiate it from a Contract Subscription object?